src/components/InvoiceForm.test.ts

```diff
@@ -0,0 +1,10 @@
+import { describe, it, expect, vi } from "vitest";
+import {render} from "@testing-library/react";
+import {InvoiceForm} from "./InvoiceForm.tsx";
+
+describe('InvoiceForm', () => {
+  it('renders without crashing', () => {
+    render(<InvoiceForm mode="view" onSubmit={vi.fn()} resolver={vi.fn()} />);
```

Comment:
I'd rather test it one level higher (EditInvoice and AddInvoice pages). But I understand this is not ready yet :)